exonum/src/storage/values.rs

```diff
@@ -282,6 +283,25 @@ impl StorageValue for DateTime<Utc> {
     }
 }
 
+/// Uses little-endian encoding.
+impl StorageValue for Duration {
+    fn into_bytes(self) -> Vec<u8> {
+        let mut buffer = vec![0; Duration::field_size() as usize];
+        let from: Offset = 0;
+        let to: Offset = Duration::field_size();
+        self.write(&mut buffer, from, to);
+        buffer
+    }
+
+    fn from_bytes(value: Cow<[u8]>) -> Self {
```

Comment:
Hmm... I think we need to reuse `Field::read` method here, don't we?

Reply:
We assume that data in storage is always valid, so unsafe should be fine here. @DarkEld3r requesting your opinion

Reply:
I think it is OK for now. We are going to the revise unsafe code after changes in serialization anyway.